.github/workflows/contributors_pull_request.yml

```diff
@@ -45,10 +45,16 @@ jobs:
         git config --global user.email "github-actions@users.noreply.github.com"
 
         git add contributors.svg
-        git commit -m "Automated deployment to update contributors.svg $(date '+%Y-%m-%d')" || exit 0
-        git push origin "${BRANCH_FROM}"
 
-        # Absolutely must download specific sha of script
-        wget https://raw.githubusercontent.com/sourcecred/sourcecred-action/9e1f0644a2398f8c99b8b19fd685566fce207959/.github/workflows/pull_request.sh
-        chmod +x pull_request.sh
-        /bin/bash pull_request.sh
+        if git diff-index --quiet HEAD --; then
```

Comment:
As a side note, the current process to update a sourcecred instance will probably still create noise. Because we're using webpack to rebuild the website. It will generate random hashes for bundle filenames. Though that should be fixed by changing how we build the website :smile:

Reply:
Definitely - at least for usrse, the updates to the contributors graphic (since there is no work on the site) are empty maybe half the time? So you are spot on - this won't be hugely impactful for the prototype, but for the contributor graphic it definitely will be. I still think it's worth having for the prototype recipe examples, because there is always the off chance that a repo is absolutely silent... !

Reply:
Agreed, and widgets was built with lower output noise in mind for usecases like this.
Have approved, just wanted to set expectations :]

Reply:
Thanks to both!

Reply:
No worries, I got the gist! There are recipes in this repository for both the contributors graphic and prototype generation, so I was able to put your comment in the correct context.